Worker.h

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) 2021 Chiu Yen-Chen, Swen Sun-Yen, Wen Yong-Wei, Yuan Wei-Chen.
+All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file. See the AUTHORS file for names of contributors.
+ */
+#ifndef SHORTLINK_SRC_WORKER_H_
+#define SHORTLINK_SRC_WORKER_H_
+#include <string>
+
+namespace shortlink
+{
+    //ThreadPool will create this class 
+    //Example:
+    // Worker worker = Worker();
+    // worker.Run("RawRequestData");
+    class Worker
+    {
+    private:
+        //worker status 0:buzy 1:free 1:Done
+        int worker_Status;
+        //parsed data 
+        std::string resp_Data;
+        //Raw data Request
+        std::string rawRequest_Data;
+
+    public:
+        Worker();
+        ~Worker();
+        //start parse data
+        void Run(const std::string& RawRequestData);
+        //return Response data
+        const std::string SendResponse();
+        //Delete this worker
+        void Delete();
```

Comment:
When do you call this? Before the destructor? Do you still have to call the destructor?